.travis.yml

```diff
@@ -1,20 +1,10 @@
 sudo: false
-
 language: php
 
-services:
-  - mongodb
-
-cache:
-  directories:
-    - $HOME/.composer/cache
-
 env:
   global:
-    - COMPOSER_ARGS="--no-interaction"
```

Comment:
@TomHAnderson  For me it is not simplification, I think we should revert it back. Sometimes we uses here `--no-scripts` or `--no-plugins` as well.

Reply:
I've restored the COMPOSER_ARGS